packages/fetchai/connections/http_server/connection.py

```diff
@@ -224,10 +224,17 @@ def from_message(cls, http_message: HttpMessage) -> "Response":
         :return: the response
         """
         if http_message.performative == HttpMessage.Performative.RESPONSE:
+
+            if http_message.is_set("headers") and http_message.headers:
+                headers = dict(email.message_from_string(http_message.headers))  # type: ignore
```

Comment:
email? cannot see where this is from...typo?; also why the type ignore. really a bad habit of yours @solarw

Reply:
email is a standart python library. please check import section.
type hinting fixed.